tsconfig.json

```diff
@@ -15,7 +15,9 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "outDir": "dist",
-    "sourceMap": true
+    "sourceMap": true,
+    "allowUnusedLabels": false,
+    "noImplicitReturns": true
```

Comment:
Remove this. I don't think we had the equivalent of this in the tslint config.

Reply:
Will do.